### PR TITLE
feat: garmin to strava

### PR DESCRIPTION
**Pull request by yihong0618**

### CODE REVIEW

**Comment by yihong0618** [comment]
@ben-29  @shaonianche can you help to review?

File: scripts/garmin_to_strava.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        )
+    )
+    loop.run_until_complete(future)
+    new_ids = future.result()
+    print(f"To upload to strava {len(new_ids)} files")
+    index = 0
+    for i in new_ids:
+        f = os.path.join(folder, f"{i}.{file_type}")
+        upload_file_to_strava(strava_client, f, file_type)
+        if i % 10 == 0:
```
[the remark this answers is not in the document]
Comment (yihong0618) [reply scripts/garmin_to_strava.py:75]: yes, my fault

[the remark this answers is not in the document]
Comment (yihong0618) [reply scripts/garmin_to_strava.py]: Fixed

**Review by ben-29 — APPROVED** [review]
Apart from the tiny mistake as commented. It's great.

I try to sync a activity, works great !! Well Done~

PS: Strava is a great app, their upload workflow even check for duplication, cool~:
image

**Comment by yihong0618** [comment]
> Apart from the tiny mistake as commented. It's great.
> 
> I try to sync a activity, works great !! Well Done~
> 
> PS: Strava is a great app, their upload workflow even check for duplication, cool~: image

Cool thanks a lot~

**Pull request merged**